README.md

```diff
@@ -155,28 +149,10 @@ functions.
     @select           select           Select
 
 
-Chaining operations is a useful way to manipulate data. There are
```

Comment:
Is this no longer supported? I haven't followed the updates on this.

Reply:
It does not work on Julia 1.0 due to Lazy.jl issues. The relevant test are also commented out. When it starts working again it should be reinserted after tests are uncommented (I have added a note there).